src/stores/store/eth1/local/local.go

```diff
@@ -406,6 +412,13 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 	return fmt.Sprintf("\x19\x01%s%s", domainSeparatorHash, typedDataHash), nil
 }
 
+// getEIP191EncodedData encodes the given message that can be later recovered
+// with the given validator. following EIP-171 spec version 0x0
+func getEIP191EncodedData(address string, msg []byte) string {
```

Comment:
MINOR: feels like `getEIP191EncodedData` could return bytes so it avoids to the transformation in the caller function.

Reply:
Yes right, but I chose that format to be consistent with existing `getEIP712EncodedData`

Reply:
Right, I will change to []byte